Update volume.md

varunturlapati opened the pull request.

balapx commented on control/volume.md:
````diff
@@ -475,6 +513,32 @@ Volume	:  970758537931791410
 ```
+For a volume created with a --group option the inspect output will reflect the flag as shown below:
+```
+sudo /opt/pwx/bin/pxctl volume inspect volFinGrp
````
volFinGrp needs to be replaced by vol1

varunturlapati replied:
Done!

varunturlapati commented:
@michaelferranti I added a new section under the CLIs but it is not showing on the left navigation bar. For ex, under CLI reference there is a section "credentials". In the repo, I added a file under control/credentials.md
Not sure what I am missing. Could you take a look please?

ferrantim commented:
@varunturlapati you just need to add the page to `home_sidebar.yml` That is the file that generates the sidebar, it is not autogenerated from the folder structure.  Just add the title and url to that file and check it locally to make sire it looks how you want.  Let me know if you have any trouble.

varunturlapati commented:
@michaelferranti - Awesome! Thanks a lot!
